src/main/java/mate/academy/internetshop/dao/impl/UserDaoImpl.java

```diff
@@ -41,4 +41,9 @@ public User update(User user) {
     public boolean delete(Long id) {
         return Storage.users.removeIf(u -> u.getId().equals(id));
     }
+
+    @Override
+    public Optional<User> findByLogin(String login) {
+        return Storage.users.stream().filter(user -> user.getLogin().equals(login)).findFirst();
```

Comment:
each stream method should be in a new line

Reply:
done